core/doc.go

```
@@ -0,0 +1,3 @@
+// Pacakge core is a set of interfaces and types that are used to implement a rollup.
```

golangci-lint (1.64.8)

`Pacakge` is a misspelling of `Package`

(misspell)